### PR TITLE
feat: init provider console with next, tailwind and shadcn

### PR DESCRIPTION
**Pull request by jigar-arc10**
- added Next js
- added tailwind CSS (merged with shared CSS)
- added shadcn

### CODE REVIEW

File: apps/provider-console/src/components/ui/dropdown-menu.tsx [hunk 1]
```diff
@@ -0,0 +1,195 @@
+"use client";
```
Comment (baktun14) [inline apps/provider-console/src/components/ui/dropdown-menu.tsx]: Please use the components in the ui package :)

Comment (jigar-arc10) [reply apps/provider-console/src/components/ui/dropdown-menu.tsx]: Yes, this is planned in the next rebase.

Comment (baktun14) [reply apps/provider-console/src/components/ui/dropdown-menu.tsx]: Ok great

File: apps/provider-console/package.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    "build": "next build",
+    "build-analyze": "set ANALYZE=true&& next build",
+    "dev": "next -p 3000",
+    "format": "prettier --write ./*.{ts,js,json} **/*.{ts,tsx,js,json}",
+    "lint": "eslint .",
+    "start": "next start",
+    "type-check": "tsc"
+  },
+  "dependencies": {
+    "class-variance-authority": "^0.7.0",
```
Comment (baktun14) [inline apps/provider-console/package.json:16]:
```suggestion
    "@akashnetwork/ui": "*",
    "class-variance-authority": "^0.7.0",
```

File: apps/provider-console/src/components/layout/Footer.tsx [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+                target="_blank"
+                rel="noreferrer"
+                className="block px-4 py-2 text-current transition-all duration-300 hover:text-primary [&>path]:fill-muted-foreground/20 hover:[&>path]:fill-primary"
+              >
+                <Youtube className="mx-auto block h-6 w-6 text-5xl" />
+              </a>
+            </li>
+            <li>
+              <a
+                href="https://github.com/akash-network/cloudmos"
```
Comment (baktun14) [inline apps/provider-console/src/components/layout/Footer.tsx]:
```suggestion
                href="https://github.com/akash-network/console"
```

File: apps/provider-console/src/components/layout/Sidebar.tsx [hunk 4]
```diff
@@ -0,0 +1,242 @@
...
+                className={cn(buttonVariants({ variant: "text", size: "icon" }), "h-8 w-8")}
+              >
+                <Youtube className="h-5 w-5" />
+                <span className="sr-only">Youtube</span>
+              </Link>
+
+              <Link
+                target="_blank"
+                rel="noreferrer"
+                href="https://github.com/akash-network/cloudmos"
```
Comment (baktun14) [inline apps/provider-console/src/components/layout/Sidebar.tsx]:
```suggestion
                href="https://github.com/akash-network/console"
```

File: package-lock.json [hunk 5]
```diff
@@ -745,6 +745,76 @@
         "typescript": "5.1.3"
       }
     },
+    "apps/landing/node_modules/clsx": {
```
Comment (baktun14) [inline package-lock.json]: The landing app is no more, so  we can remove these.

**Review by baktun14 — CHANGES_REQUESTED** [review]
Minor changes

**Comment by jigar-arc10** [comment]
> Minor changes

Fixed all the minor changes.

**Review by baktun14 — APPROVED** [review]

**Pull request merged**